### PR TITLE
Start at a "getting started" page in the intro section.

### PR DESCRIPTION
**Pull request by icarusz**
Topper, its far from perfect but at least walks a player through the really 101 to get started.
Plan to add a "CC for the crypto noob" next.

### CODE REVIEW

**Comment by tobowers** [comment]
@icarusz broken build here.

File: docs/firstbet.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Crypto Colosseum is (in the beginning) a simple gambling game.
+
+Gladiators have tournaments.  You place your bet on which of the four gladiators will win the tournament.  You get a share of the prize purse if you are right.
+
+To get started the first thing you need to do is sign up.  Click on the orange "Sign in / Sign up" in the upper right hand corner to do so. Its easy we promise.
+
+![image](signin.png)
+
+We have provided several different ways for you to do so but if you are new to the game and to crypto, the easiest way is to use an existing account such as gmail.  When you do this we are actually creating a special vault or wallet for you to recieve and spend your crypto currency (PTG) from.  (As well as other things.)  You own it through your gmail or social media account we can not access it without your permission.
```
Comment (tobowers) [inline docs/firstbet.md:15]:
```suggestion
There are several different ways to login, but if you are new to the game and to crypto, the easiest way is to use an existing account (like gmail).  When you do this we are actually creating a special vault or wallet to receive and spend your crypto currency (PTG).  (As well as hold your in-game NFTs)  You own it and we can not access it without your permission.
```

File: docs/firstbet.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+![image](picktraditional.png)
+
+The creation of your wallet and the linking to your existing account elsewhere is done through Torus.  You don't need to worry about that for now but you can access that through the "T" in the bottom left at all times.
```
Comment (tobowers) [inline docs/firstbet.md:19]: link to torus?

File: docs/firstbet.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Now the last step is to place a bet on the tournament page.  There is a lot on the tournament page but all you need to worry about is which gladiator you want to bet on.  Pick one and find the associated orange "Bet Now" button and click it.  Below you can see what some of the other info is but for your first bet, bet small (maybe 100 PTG) and pick whichever appeals to you the most.
+
+![image](tournament.png)
+
+If you are the type to want to know more before you bet check out [combat basiscs](https://docs.cryptocolosseum.com/gamemechanics/combatexample) to learn more.
+
+Once you click to place your bet a popup with ask you how much to bet.  Then once you submit that your wallet will check if you are really sure you want to put some of the PTG in your pocket on the line.  Confirm that and you are off and running!  (Look for popup blockers etc if you dont see the confirmation requests).
+
+The game holds surprising complexity and there are many aspects to learn but bet a few times and then explore the item market (on bonding curves), bidding on a gladiator contract (all income generating NFTs) and even explore getting your own share of the factions and their winnings.
```
Comment (tobowers) [inline docs/firstbet.md:41]:
```suggestion
The game holds surprising complexity and there are many aspects to learn, but bet a few times and then explore the item market (on bonding curves), bidding on a gladiator contract (income generating NFTs) and even explore getting your own share of the factions and their winnings.
```

File: docs/firstbet.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Most of all have fun!  
+Its whimsical violence.  Its degengerate gambling and we just gave you free money.
```
Comment (tobowers) [inline docs/firstbet.md:44]:
```suggestion
It's whimsical violence.  It's degengerate gambling and The Arena just gave you free money.  
```

File: docs/firstbet.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+If you are the type to want to know more before you bet check out [combat basiscs](https://docs.cryptocolosseum.com/gamemechanics/combatexample) to learn more.
+
+Once you click to place your bet a popup with ask you how much to bet.  Then once you submit that your wallet will check if you are really sure you want to put some of the PTG in your pocket on the line.  Confirm that and you are off and running!  (Look for popup blockers etc if you dont see the confirmation requests).
+
+The game holds surprising complexity and there are many aspects to learn but bet a few times and then explore the item market (on bonding curves), bidding on a gladiator contract (all income generating NFTs) and even explore getting your own share of the factions and their winnings.
+
+Most of all have fun!  
+Its whimsical violence.  Its degengerate gambling and we just gave you free money.  
+Party time!
```
Comment (tobowers) [inline docs/firstbet.md:45]:
```suggestion
Battle time!
```

**Review by tobowers — CHANGES_REQUESTED** [review]
couple of little typos overall a really good start.

File: docs/firstbet.md (hunk 2 above)
Comment (icarusz) [reply docs/firstbet.md]: Done.

**Pull request merged**